mod/profile_photo.php

```diff
@@ -221,7 +218,7 @@ function profile_photo_content(App $a)
 				Worker::add(PRIORITY_LOW, "Directory", $url);
 			}
 
-			goaway(System::baseUrl() . '/profiles');
+			goaway($url);
```

Comment:
Where is `$_SESSION[ 'my_url']` set and couldn't it be reconstructed instead of being stored in session?

Reply:
Maybe, I was just using whats there. Have no idea how its set...

Reply:
Few things are called “my_url” in the project. If you don’t know how to search the whole project on your computer, you can do it in GitHub in the search bar at the top of the desktop website.

Reply:
_I'm a 40 year old Linux-C-Developer. Just for further reference and as a soft of introduction. :)_

I have searched (grep'ed!) for where the session parameter is set but I'm unclear where that might be. I have changed that to the same construct that is used everywhere else.